new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/ConfigMap.java

```diff
@@ -15,20 +16,35 @@
    * @param configMap V1ConfigMap object containing config map configuration data
    * @return true on success
    * @throws ApiException if Kubernetes client API call fails
-   */
+  */
   public static boolean create(V1ConfigMap configMap) throws ApiException {
     return Kubernetes.createConfigMap(configMap);
   }
 
   /**
-   /**
    * Delete Kubernetes Config Map.
    *
    * @param name name of the Config Map
    * @param namespace name of namespace
    * @return true if successful, false otherwise
-   */
+  */
   public static boolean delete(String name, String namespace) {
     return Kubernetes.deleteConfigMap(name, namespace);
   }
+
+  /**
+   * List Kubernetes Config Map in a namesapce.
+   *
+   * @param namespace name of namespace
+   * @return List of Config Maps in a namespace
+  */
+  public static V1ConfigMapList list(String namespace) {
+    V1ConfigMapList retConfigMapList = null;
+    try {
+      retConfigMapList = Kubernetes.listConfigMaps(namespace);
+    } catch (ApiException api) {
```

Comment:
Don't catch the exception here. Just return whatever Kubernetes.listConfigMaps(namespace) returns. The test needs to deal with the exception.

Reply:
Done.  Modified the impl/Kubernetes.java accordingly